src/platform/windows/display_vram.cpp

```diff
@@ -938,6 +953,10 @@ namespace platf::dxgi {
       return false;
     }
 
+    if (t.Width != t.Height) {
+      BOOST_LOG(error) << "NOT SQUARE";
```

Comment:
If this isn't just a debugging artifact that will be removed, change the message to something better.

Reply:
Oops. my bad.